workers/tests/unittests/test_config.py

```diff
@@ -0,0 +1,102 @@
+import inspect
+import os
+import tempfile
+import unittest
+from unittest import mock
+
+from ohsome_quality_analyst.config import config
+
+
+class TestConfig(unittest.TestCase):
+    @mock.patch.dict("os.environ", {}, clear=True)
+    def test_get_config_path_empty_env(self):
+        self.assertEqual(
+            config.get_config_path(),
+            os.path.join(
+                os.path.dirname(
+                    inspect.getfile(config),
+                ),
+                "config.yaml",
+            ),
+        )
+
+    @mock.patch.dict(
+        "os.environ",
+        {"OQT_CONFIG": os.path.join(tempfile.gettempdir(), "config.yaml")},
+        clear=True,
+    )
+    def test_get_config_path_set_env(self):
+        self.assertEqual(
+            config.get_config_path(),
+            os.path.join(tempfile.gettempdir(), "config.yaml"),
```

Comment:
see above. It shouldn't be *exactly* the same implementation. I would use strings here.

Reply:
Okay maybe I just don't get why you're using the temp directory and not just any string/path?